backend/src/main/java/com/ambacam/rest/roles/RolesResource.java

```diff
@@ -29,17 +29,14 @@
 
 	@Autowired
 	private RoleService roleService;
-	@Autowired
-	RoleResource roleResource;
 
 	@RequestMapping(method = RequestMethod.POST)
 	@Consumes(MediaType.APPLICATION_JSON)
 	@Produces(MediaType.APPLICATION_JSON)
-	public Role create(@RequestBody @Valid Role entity) {
+	public Role create(@RequestBody @Valid Role create) {

```

Comment:
Concerning the name of the incoming object, I used to keep the name of the class as variable name. Eg
`public Role create(@RequestBody @Valid Role role) {}`
What do you think ?

Reply:
That is right, I agree with you @Coccoonx . I think, it is a good practice.